tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="cp319" name="Cellprofiler" version="@CP_VERSION@">
+    <description>Run a CellProfiler pipeline</description>
+    
+    <macros>
+        <import>macros.xml</import>
+        <xml name="test_assert_content" token_n="291">
+            <assert_contents>
+                <has_n_lines n="@N@" />
+            </assert_contents>
+        </xml>
+    </macros>
+
+    <expand macro="requirements" version="@CP_VERSION@" package="cellprofiler" />
```

Comment:
I think here you are overdoing the macros in macros ;)
Just create one macro for python and the other for cellprofiler :)